widget/slider.go

```diff
@@ -183,6 +199,7 @@ func (s *Slider) TypedKey(key *fyne.KeyEvent) {
 			s.SetValue(s.Value + s.Step)
 		}
 	}
+	s.checkInvokeOnChangeEnded()
```

Comment:
This should be moved to the SetValue function, since that function calls OnChanged, and should call OnChangeEnded as well if the value is updated. Will push a fix.